components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/PreferenceRetrievalClient.java

```diff
@@ -98,6 +99,18 @@ public boolean checkSelfRegistrationLockOnCreation(String tenant) throws Prefere
         return checkPreference(tenant, SELF_SIGN_UP_CONNECTOR, SELF_SIGN_UP_LOCK_ON_CREATION_PROPERTY);
     }
 
+    /**
+     * Check send confirmation on account creation is enabled or not.
+     *
+     * @param tenant tenant domain name.
+     * @return returns true if send confirmation on creation is enabled.
+     * @throws PreferenceRetrievalClientException
```

Comment:
complete the comment

Reply:
Addressed with commit a84bf356ff84ef39313f2fd69d596fe09eeb35e4